content/en/deployment/strategies/blue-green.md

```diff
@@ -88,3 +106,50 @@ Monitor the progress and **Approve & Continue** or **Roll back** the blue/green
 
 
 
+## Optional configuration
+
+### Preview service
+You can also create a `previewService` Kubernetes Service object so you can programmatically or manually observe the new version of your software before exposing it to traffic via the `activeService`. This is the `trafficManagement.kubernetes.previewService` field in the YAML configuration.
```

Comment:
Suggesting a rewrite of what is written in this section.

With Blue/Green strategy, you can programmatically or manually observe the new version of your software before exposing it to traffic. You can accomplish this by defining a previewService in the deployment configuration in the trafficManagement block.

previewService is the name of a Kubernetes Service object, that points to the deployment object. This service needs to exist on your cluster, or can be deployed alongside your application with CD-as-a-Service. The previewService must be a different service than the one used for activeService. 
When creating the Replicaset for the new version, CD-as-a-Service will inject the labels into preview service, so that the preview Service points to the new version of application. q

Reply:
There are 2 ways to expose the preview service, 
1. (Recommended) You can use the `exposeService` step along with a `pause` that waits for manual approval in `redirectTrafficAfter` section (Not sure if there is documentation for this thing or not)
2. You can define a custom address in the Kubernetes Ingress for the service. CD-as-a-Service will read the Ingress in the cluster and namespace it is deploying to and will expose the link, if available. (This was true, when it was developed. Does it still work? idk)